databuilder/models/column_model.py

```diff
@@ -7,7 +7,7 @@
 from databuilder.models.table_metadata import TableMetadata, DESCRIPTION_NODE_LABEL
 
 
-class TestColumnMetadata(Neo4jCsvSerializable):
+class ColumnMetadata(Neo4jCsvSerializable):
```

Comment:
there is an existing metadata model(https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/models/table_metadata.py#L34), we can't rename it as it is used in test.

Reply:
ok, I was not aware of it. I'm not sure what the purpose of the previous test models was then...

Reply:
yeah, we should remove those test model and use the existing(or make existing models work for quick start), forgot why we create those for quick start

Reply:
@feng-tao I think I understand why they exist now. From what I can tell, table_metadata.py contains both ColumnMetadata and TableMetadata objects, and it does not appear possible to create a column without a table with this class. However, with the former TestColumnMetadata class, you could, because it has the extra fields in it to bind it to the table. 

So, I'm not sure we can create columns from simple csv files in this way (using table_metadata.py that is). I think we would either need to:

1. keep TestColumnMetadata but rename it to something to indicate it is for standalone, or csv loading (simplest solution)
2. move from csv to a hierarchical form (e.g. json) for file based loading (would take a little more time)

I think I'd prefer to start with 1, to unblock and then put another issue up to move file based loading to json next. Unless there is another solution here, that I'm missing. wdyt?

Reply:
@feng-tao I've implemented this change as well as a few others, added a test for csv parsing, etc. All checks are passing now. Can you take another look please?